scripts/convert.py

```diff
@@ -0,0 +1,244 @@
+# Standard library
+from os import path, walk, remove, makedirs
+
+# Third-party
+from astropy import log as logger
+logger.setLevel('INFO')
+
+from nbconvert.preprocessors import ExecutePreprocessor, CellExecutionError
+from nbconvert.exporters import RSTExporter
+from nbconvert.writers import FilesWriter
+import nbformat
+
+IPYTHON_VERSION = 4
+
+class NBConverter(object):
```

Comment:
Maybe call this ``NBSphinxTutorialsConverter`` or something?  Right now it looks like it's part of `nbconvert` itself or something